StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/build/StyleCop.Analyzers.targets

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
❗️ This needs to be **StyleCop.Analyzers.props**.

Reply:
If this is a `.props` file, it is imported _before_ the project items are declared.  This would make it impossible to move the `None` item to `AdditionalFiles`.  So I used `.targets` so the items are imported after the project items.

Reply:
It's important that the AdditionalFiles items be added in a .props file (before the project file is evaluated), or it becomes impossible to use `Update` functionality inside the project file to modify the included element.